manifests/x/Xiaoe/Xiaoetong/1.10.13/Xiaoe.Xiaoetong.yaml

```diff
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.2.1 $debug=QUSU.CRLF.7-3-0.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: Xiaoe.Xiaoetong
+PackageVersion: 1.10.13
+DefaultLocale: en-US
```

Comment:
should en-US be default, or should zh-CN be default?

Reply:
I set en-US as default so that users using neither en-US nor zh-CN will finnally receive en-US metadata.